Library/Formula/ola.rb

```diff
@@ -20,14 +21,15 @@ class Ola < Formula
 
   option :universal
   option "with-ftdi", "Install FTDI USB plugin for OLA."
+  option "with-rdm-tests", "Install RDM Tests for OLA."
 
   depends_on "pkg-config" => :build
   depends_on "cppunit"
   depends_on "protobuf-c"
   depends_on "libmicrohttpd"
-  depends_on "libusb"
-  depends_on "liblo"
   depends_on "ossp-uuid"
+  depends_on "libusb" => :optional
```

Comment:
I've made both these optional, but I'm wondering if they should be recommended? They enable plugins for the specific devices, e.g. for OSC support. I imagine most people don't use OSC, but should we enable it by default (recommeded) so it's there and "it just works"? I know other projects e.g. Debian have policies on this.